CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Views/NotificationPopupPage.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using CheckDrive.Mobile.ViewModels;
+using Rg.Plugins.Popup.Pages;
+using System;
+using Xamarin.Forms.Xaml;
+
+namespace CheckDrive.Mobile.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class NotificationPopupPage : PopupPage
+    {
+        private readonly ChatViewModel _chatViewModel;
+        public NotificationPopupPage(string massage)
+        {
+            InitializeComponent();
+
+            _chatViewModel = new ChatViewModel();
+            _chatViewModel.Message = massage; 
```

Comment:
maybe it's better to pass message as a parameter rather than changing it separately?